### PR TITLE
Add long training with pytorch lightning example

### PR DESCRIPTION
**Pull request by yirenlu92**
Add pytorch lightning (long training) example

### CODE REVIEW

**Review by erikbern — COMMENTED** [review]
Looks great!

File: 06_gpu_and_ml/long-training/train.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import lightning as L
```
Comment (charlesfrye) [inline 06_gpu_and_ml/long-training/train.py:1]:
```suggestion
---
lambda-test: false
---
import lightning as L
```

File: 06_gpu_and_ml/long-training/train.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# ---
+# lambda-test: false
```
Comment (charlesfrye) [inline 06_gpu_and_ml/long-training/train.py:2]:
```suggestion
# lambda-test: false
# pytest: false
```

File: 06_gpu_and_ml/long-training/long-training.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
+# ---
+# cmd: ["modal", "run", "06_gpu_and_ml.long-training.long-training", "--detach"]
```
Comment (charlesfrye) [inline 06_gpu_and_ml/long-training/long-training.py:2]:
```suggestion
# cmd: ["modal", "run", "--detach", "06_gpu_and_ml.long-training.long-training"]
```

**Review by charlesfrye — APPROVED** [review]
nice work!

**Pull request merged**